packages/ngx-flicking/projects/ngx-flicking/package.json

```diff
@@ -3,8 +3,8 @@
   "version": "4.2.3",
   "description": "Everyday 30 million people experience. It's reliable, flexible and extendable carousel.",
   "peerDependencies": {
-    "@angular/common": "^7.2.0",
-    "@angular/core": "^7.2.15"
+    "@angular/common": ">=11.0.0",
```

Comment:
It seems that all versions 7, 8, 9, 10, and 11 should be included.

Reply:
I've excluded 7~9 as it's not supported as LTS in Angular community.
I wish to it be 10+, but the current version works only in the 11+, maybe I should recreate the whole project to support that.

Also, I'm wondering that if the library supports Angular major 7, does that also mean the other versions(8+)?
I'm worried that it might not support those latest versions if I built it with Angular 7